CIP-0035/README.md

```diff
@@ -0,0 +1,38 @@
+---
+CIP: 35
+Title: Daedalus URI Scheme
+Authors: Daniel Main <daniel.main@iohk.io>
+Comments-URI:
+- https://github.com/cardano-foundation/CIPs/pull/130
+Status: Draft
+Type: Informational
+Created: 2022-03-18
+License: CC-BY-4.0
+---
+
+# Abstract
+
+This proposal is an extension of CIP-0013, which describes a basic URI scheme to handle ada transfers or partial transactions with Daedalus or any desktop Wallet installed in the operating system.
+
+# 💡 Motivation
+
+DEX, Web-dApps and Desktop-dApps (e.g. games or desktop applications) should have the possibility to interact directly with desktop wallets. By defining a URI scheme and registering it in the operating system, it will allow opening Daedalus or any desktop wallet same as when you call or click a mailto:email@sample.com and your email client is opened.
```

Comment:
changing this to `mailto:email@sample.com` (i.e, adding backticks around it) should prevent the email address from being formatted differently (as a clickable & invalid email address) in the formatted Markdown (as it appears now).